### PR TITLE
Restaurants list branch

### PR DESCRIPTION
**Pull request by bulic**

### CODE REVIEW

**Comment by stanislavgrujic** [comment]
@vrackovic Could you please have a look at this?

File: client-app/src/app/components/restaurant-list/restaurant-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { RestaurantService } from "src/app/services/restaurant.service";
+import { Subscription } from "rxjs";
+
+@Component({
+  selector: "app-restaurant-list",
+  templateUrl: "./restaurant-list.component.html",
+  styleUrls: ["./restaurant-list.component.scss"],
+})
+export class RestaurantListComponent implements OnInit, OnDestroy {
+  // @Input() id: number;
```
Comment (vrackovic) [inline client-app/src/app/components/restaurant-list/restaurant-list.component.ts:12]: Please delete commented code

File: client-app/src/app/models/restaurant.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  name: string;
+  email?: string;
+  logoUrl?: string;
+  description?: string;
+  contact?: string;
+  isActive?: boolean;
+  dateTime?: string;
+  sheetId?: string;
+  // public List<MealDto> Meals { get; set; }
+  // public List<DailyMenuDto> DailyMenus { get; set; }
```
Comment (vrackovic) [inline client-app/src/app/models/restaurant.ts:12]: Please delete commented code

File: Exebite.GoogleSheetAPI/Properties/Resources.resx [hunk 3]
```diff
@@ -136,6 +136,6 @@
     <value>--</value>
   </data>
   <data name="TOPLI_OBROK" xml:space="preserve">
-    <value>1E3tzbemuhLgNUpVuN11Gr1yeKEIgOFN7q2z7UBLnAt8</value>
+    <value>1Oo7AcHQS3rs-PSRaZyAbm_bH8s8dmkHcGg3Ji2dqkS4</value>
   </data>
 </root>
```
Comment (vrackovic) [inline Exebite.GoogleSheetAPI/Properties/Resources.resx:141]: Please add new line at file end.

File: client-app/src/app/components/filter-view-by/filter-view-by.component.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+        </div>
+    </div>
+    <div class="col-11">
+        <div class="btn-group btn-group-sm" role="group" aria-label="First group">
+            <button type="button" class="btn btn-outline-dark mr-3 rounded-pill text-uppercase"
+                [ngClass]="{ 'active' : isValue == 1 }" (click)="showRestorauntList()">restaurant</button>
+            <button type="button" class="btn btn-outline-dark rounded-pill text-uppercase">price</button>
+        </div>
+    </div>
+</div>
```
Comment (vrackovic) [inline client-app/src/app/components/filter-view-by/filter-view-by.component.html:14]: Please add new line at file end.

File: client-app/src/app/components/header/header.component.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+                        class="caret"></span></a>
+                <ul class="dropdown-menu">
+                    <li class="dropdown-item text-center"><a (click)="signOutNavigate()"
+                            style="cursor: pointer;">Logout</a>
+                    </li>
+                </ul>
+            </li>
+        </ul>
+    </div>
+</nav>
```
Comment (vrackovic) [inline client-app/src/app/components/header/header.component.html:20]: Please add new line at file end.

File: client-app/src/app/components/home/home.component.html [hunk 6]
```diff
@@ -1,10 +1,20 @@
...
+                <div>
+                    <app-filter-view-by></app-filter-view-by>
+                </div>
+                <div>
+                    <app-restaurant-list></app-restaurant-list>
+                </div>
+            </div>
+        </div>
     </div>
 </div>
```
Comment (vrackovic) [inline client-app/src/app/components/home/home.component.html:20]: Please add new line at file end.

File: client-app/src/app/components/login/login.component.html [hunk 7]
```diff
@@ -1,5 +1,12 @@
...
+        <div class="col-12 text-center">
+            <img src="../../../assets/ExeBite logo placeholder 2.png" class="rounded" alt="Exebite">
+            <div class="row">
+                <div class="col-12 text-center">
+                    <button class="btn btn-danger rounded-pill pl-5 pr-5" (click)="signInWithGoogle()">Login</button>
+                </div>
+            </div>
+        </div>
     </div>
 </div>
```
Comment (vrackovic) [inline client-app/src/app/components/login/login.component.html:12]: Please add new line at file end.

File: client-app/src/app/components/restaurant-detail/restaurant-detail.component.html [hunk 8]
```diff
@@ -0,0 +1,5 @@
+<div *ngIf="foodsData">
+    <div>
+        <div *ngFor="let food of foodsData">{{food.name}}</div>
+    </div>
+</div>
```
Comment (vrackovic) [inline client-app/src/app/components/restaurant-detail/restaurant-detail.component.html:5]: Please add new line at file end.

File: client-app/src/app/components/restaurant-list/restaurant-list.component.html [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+            routerLinkActive="active">
+            <div>
+                <h5>{{item.name}}</h5>
+            </div>
+        </a>
+    </li>
+</ul>
+<div>
+    <router-outlet></router-outlet>
+</div>
```
Comment (vrackovic) [inline client-app/src/app/components/restaurant-list/restaurant-list.component.html:13]: Please add new line at file end.

File: client-app/src/index.html [hunk 10]
```diff
@@ -1,13 +1,20 @@
...
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
 </head>
+
 <body>
   <app-root></app-root>
 </body>
-</html>
+
+</html>
```
Comment (vrackovic) [inline client-app/src/index.html:20]: Please add new line at file end.

**Pull request merged**